packages/camera/camera/test/camera_value_test.dart

```diff
@@ -13,26 +13,28 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import 'camera_preview_test.dart';
+
 void main() {
   group('camera_value', () {
     test('Can be created', () {
       const CameraValue cameraValue = CameraValue(
-        isInitialized: false,
-        previewSize: Size(10, 10),
-        isRecordingPaused: false,
-        isRecordingVideo: false,
-        isTakingPicture: false,
-        isStreamingImages: false,
-        flashMode: FlashMode.auto,
-        exposureMode: ExposureMode.auto,
-        exposurePointSupported: true,
-        focusMode: FocusMode.auto,
-        deviceOrientation: DeviceOrientation.portraitUp,
-        lockedCaptureOrientation: DeviceOrientation.portraitUp,
-        recordingOrientation: DeviceOrientation.portraitUp,
-        focusPointSupported: true,
-        previewPauseOrientation: DeviceOrientation.portraitUp,
-      );
+          isInitialized: false,
+          previewSize: Size(10, 10),
+          isRecordingPaused: false,
+          isRecordingVideo: false,
+          isTakingPicture: false,
+          isStreamingImages: false,
+          flashMode: FlashMode.auto,
+          exposureMode: ExposureMode.auto,
+          exposurePointSupported: true,
+          focusMode: FocusMode.auto,
+          deviceOrientation: DeviceOrientation.portraitUp,
+          lockedCaptureOrientation: DeviceOrientation.portraitUp,
+          recordingOrientation: DeviceOrientation.portraitUp,
+          focusPointSupported: true,
+          previewPauseOrientation: DeviceOrientation.portraitUp,
+          description: FakeController.fakeDescription);
```

Comment:
Nit: Please add the trailing comma, for consistency with the opposite change below.